syntheseus/reaction_prediction/environments/environment_shared.yml

```diff
@@ -0,0 +1,12 @@
+name: syntheseus-single-step
+channels:
+  - defaults
+  - conda-forge
+  - pytorch
+dependencies:
+  - numpy
+  - pandas
+  - pip
+  - python==3.9.7
+  - pytorch=1.10.2=py3.9_cuda11.3_cudnn8.2.0_0
+  - rdkit=2021.09.4
```

Comment:
The pin may be unnecessary; I'm rerunning the evaluation of all models to see if everything is fine when the CUDA version is unpinned.

Reply:
Do you know how to unpin this in a way that works reliably? I tried truncating to `pytorch=1.10.2`, but by default that gets me the CPU-only version. I tried adding `cudatoolkit` to the list of dependencies, but when I start with a docker image that has CUDA 11.3, adding a `cudatoolkit` without a specific version gets me `cudatoolkit==11.8`. Even if I fix the version for `cudatoolkit` I still end up getting the CPU version of `pytorch`... 😕

(And if we cannot find a good solution, perhaps we can leave the explicit pin and make a note that this can be changed if desired?)